### PR TITLE
[Enhancement] Panorama - reputation command to return a list of results

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
epic: https://github.com/demisto/etc/issues/29016

## Description
* Use the new `CommandResults.indicator` parameter.
* Update the commands **url**, **panorama-get-url-category**, **panorama-get-url-category-from-cloud** and **panorama-get-url-category-from-host**  to return a different result for each indicator.

## Does it break backward compatibility?
   - [x] Yes
       - Further details: custom scripts that run these commands using `executeCommand` might miss the switch, and handle only the first result.

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/PAN-OS/ReleaseNotes/1_6_5.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+**Breaking Change** **url**, **panorama-get-url-category**, **panorama-get-url-category-from-cloud** and **panorama-get-url-category-from-host**
```
Comment (amshamah419) [inline Packs/PAN-OS/ReleaseNotes/1_6_5.md:4]:
```suggestion
**Breaking Change:** ***url***, ***panorama-get-url-category***, ***panorama-get-url-category-from-cloud*** and ***panorama-get-url-category-from-host*** 
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -3542,17 +3479,17 @@ def panorama_edit_edl(edl_name, element_to_change, element_value):
...
-    element_to_change = demisto.args()['element_to_change']
-    element_value = demisto.args()['element_value']
+    edl_name = args['name']
+    element_to_change = args['element_to_change']
+    element_value = args['element_value']
 
     result, edl_output = panorama_edit_edl(edl_name, element_to_change, element_value)
 
-    demisto.results({
+    return_results({
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3492]: are you sure this is legit?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -3583,18 +3520,16 @@ def panorama_delete_edl(edl_name):
...
     """
-    edl_name = demisto.args()['name']
-
     edl = panorama_delete_edl(edl_name)
     edl_output = {'Name': edl_name}
     if DEVICE_GROUP:
         edl_output['DeviceGroup'] = DEVICE_GROUP
 
-    demisto.results({
+    return_results({
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3532]: are you sure this is legit?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 4]
```diff
@@ -3637,22 +3572,21 @@ def panorama_refresh_edl(edl_name: str, edl_type: str, location: str, vsys: str)
...
-    edl_name = args.get('name')
+    edl_name = args.get('name', '')
     edl_type = args.get('edl_type', '')
     location = args.get('location', '')
     vsys = args.get('vsys', '')
 
     result = panorama_refresh_edl(edl_name, edl_type, location, vsys)
 
-    demisto.results({
+    return_results({
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3589]: are you sure this is legit?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 5]
```diff
@@ -5265,14 +5195,14 @@ def panorama_download_panos_status(target: str, job_id: str):
...
+def panorama_download_panos_status_command(args: dict):
     """
     Check jobID of panos download status
     """
     if DEVICE_GROUP:
         raise Exception('PAN-OS version download status is only supported on Firewall (not Panorama).')
-    target = str(demisto.args()['target']) if 'target' in demisto.args() else None
-    job_id = demisto.args()['job_id']
+    target = str(args['target']) if 'target' in args else None
+    job_id = args['job_id']
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:5205]: maybe use a safe get?

**Review by bakatzir — COMMENTED** [review]
~~some of the `return_results` which do not return md only, but `Type` `Contents` and more are misused, i've commented only on few.~~

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 2 above)
Comment (DeanArbel) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3492]: yeah, also required (warning level) by sdk lint

Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3492]: you are correct, seen it just now.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 5 above)
Comment (DeanArbel) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:5205]:
```suggestion
    job_id = args.get('job_id')
```

**Review by yuvalbenshalom — APPROVED** [review]

**Review by bakatzir — APPROVED** [review]
very nice work handling all the lint issues we had in there.

**Pull request merged**